### PR TITLE
FRCUserProgram is now given the CAP_SYS_NICE capability so it can set real-time priorities

### PR DESCRIPTION
**Pull request by calcmogul**
Since child processes can't inherit CAP_SYS_NICE from setuid processes, this commit adds an SSH command that invokes setcap instead. This works on recent kernels.

### CODE REVIEW

**Comment by JLLeitschuh** [comment]
So what does this do in layman's terms?

**Comment by calcmogul** [comment]
It allows instances of the Task class to change their real-time thread priority.

**Comment by PeterJohnson** [comment]
@frcjenkins test this please

**Review by PeterJohnson — APPROVED** [review]
